crates/criticaltrust/src/integrity/verifier.rs

```diff
@@ -142,7 +142,7 @@ impl<'a> IntegrityVerifier<'a> {
                 inner: e,
             })?
             .signed
-            .into_verified(self.keychain)
+            .into_verified(self.keychain, self.keychain.revocation_info())
```

Comment:
I think I see one of those plays you mentioned being unhappy with the code. 🤔

Reply:
I think I am going to leave this be because of the wide area cast by the trait object on the receiver side but this was a good catch!